test/framework/test_framework.py

```diff
@@ -340,3 +340,32 @@ def circuits(self):
         }
 
         self.assertEqual(exp.job_info(backend=backend), job_info)
+
+    @ddt.data((None, "Test1"), (None, "Test2"))
+    @ddt.unpack
+    def test_experiment_type(self, exp_type1, exp_type2):
+        """Test the experiment_type setter for the experiment."""
+
+        class MyExp(BaseExperiment):
+            """Some arbitrary experiment"""
```

Comment:
Check if `FakeExperiment` can be used here.

Reply:
doesn't work 
```
TypeError: FakeExperiment.__init__() got an unexpected keyword argument 'experiment_type'
```

Reply:
Ok